packages/app-bridge/sonar-project.properties

```diff
@@ -2,5 +2,5 @@ sonar.projectKey=frontify_brand-sdk_app-bridge
 sonar.organization=frontify
 
 sonar.javascript.lcov.reportPaths=./coverage/lcov.info
-sonar.coverage.exclusions=src/tests/**/*,src/types/**/*,src/**/*.spec.ts,src/**/*.spec.tsx
+sonar.coverage.exclusions=src/tests/**/*,src/types/**/*,src/**/*.spec.ts,src/**/*.spec.tsx,src/AppBridgeBlock.ts
```

Comment:
@SamuelAlev not sure if this is the way to go but as `AppBridgeBlock` is just an interface, it makes no sense to test it, yet its counted in for test coverage

Reply:
It can be `src/AppBridge*.ts` as others are interfaces as well. Also, we might have to move this change to a separate PR as the Sonarlint configuration for the tests is always picked from the main branch.

Reply:
Optimally the interface is tested ([docs](https://vitest.dev/api/expect-typeof.html)) to prevent regression.
So let's not ignore it, I will just force the merge.